src/state/opamSysInteract.ml

```diff
@@ -130,6 +130,14 @@ let family =
   ) in
   fun () -> Lazy.force family
 
+let yum_cmd = lazy begin
```

Comment:
Just cosmetic and personal appreciation: why not `yum_cmd: unit -> string` with an inner lazy/force?

Reply:
I’m not sure why it would bring over just `string lazy_t`